network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -48,13 +48,14 @@ <h1 class="tw-h1-heading">{{ page.title }}</h1>
 
 <div class="container">
   <div class="row">
-    <div class="pt-1 pt-md-5 pb-4 pb-md-5 col-lg-8 m-auto bottom-share-button-container">
-      {% include "./fragments/share-buttons.html" with page=page share_text=share_text_translated %}
```

Comment:
Hi @fessehaye! Wanted to note that it was requested to remove these square share buttons from the bottom of the page, So I am doing that in this PR. Thanks!